### ISSUE TITLE
Coulnd't deploy with docker-compose `Unsupported config option for services: 'firefly_iii_db'`

### ISSUE THREAD

**Issue by Maxiride** [firefly-iii/firefly-iii#2312]
I'm on a Raspberry Pi 3 B+ (armhf), I've downloaded the `docker-compose.yml` file and without changing any line I've tried to deploy it but I've received this error:

```
pi@raspberrypi:~/firefly $ docker-compose up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services: 'firefly_iii_db'
Unsupported config option for volumes: 'firefly_iii_db'
Unsupported config option for networks: 'firefly_iii_net'
```

First thing first the error thrown (even with --verbose) doesn't actually point out the wrong option so I can't even help myself there D:

I've doublechecked the docker-compose.yml file for indentation errors without success. 

So if possible I'm looking for ideas to debug the issue.

```
Docker and Docker Compose versions:
pi@raspberrypi:~/firefly $ docker-compose -v
docker-compose version 1.8.0, build unknown
pi@raspberrypi:~/firefly $ docker -v
Docker version 18.09.0, build 4d60db4
pi@raspberrypi:~/firefly $
```

**Comment by Maxiride** [comment]
I updated to the latest version of docker-compose and everything went fine. I suppose the issue was using a previous version otherwise I have no clue about the issue.

For anyone else stumbling here, I opted for the [hypriot](https://github.com/hypriot/image-builder-rpi) OS which is better optimized for docker usage. Raspbian, Ubuntu etc for arms have unmaintained and old repos (like the docker-compose one).
